packages/stark-ui/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./public_api";
```

Comment:
Those files have been removed when we implemented ng-packagr.
You can the link in showcase as we do in `stark-core` or `stark-ui` : 
```json
"@nationalbankbelgium/stark-ui": ["../packages/stark-ui/public_api.ts"],
"@nationalbankbelgium/stark-ui/testing": ["../packages/stark-ui/testing/public_api.ts"],
```

Reply:
Oke, that works. 👍 
The environment doesn't give me an issue anymore either, but It should be relative right? 🤔